plugins/com.google.cloud.tools.eclipse.appengine.login/plugin.properties

```diff
@@ -1,2 +1,6 @@
 pluginName=Google Cloud Platform Login Support
 providerName=Google, Inc.
+
+appEngineTopLevelMenuLabel=App Engine
+loginMenuItemLabel=Login to Google...
+loginToolbarTooltip=Click to login to Google
```

Comment:
tooltips should provide additional info. How about "Authorize Eclipse to operate on Google Cloud Platform resources."